src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/OOPTagHelperResolver.cs

```diff
@@ -73,69 +72,50 @@ public override async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project
                 TagHelperResolutionResult result = null;
                 if (factory != null)
                 {
-                    result = await ResolveTagHelpersOutOfProcessAsync(factory, workspaceProject, projectSnapshot).ConfigureAwait(false);
+                    result = await ResolveTagHelpersOutOfProcessAsync(factory, workspaceProject, projectSnapshot, cancellationToken).ConfigureAwait(false);
                 }
 
                 if (result == null)
                 {
                     // Was unable to get tag helpers OOP, fallback to default behavior.
-                    result = await ResolveTagHelpersInProcessAsync(workspaceProject, projectSnapshot).ConfigureAwait(false);
+                    result = await ResolveTagHelpersInProcessAsync(workspaceProject, projectSnapshot, cancellationToken).ConfigureAwait(false);
                 }
 
                 return result;
             }
             catch (Exception exception)
             {
-                throw new InvalidOperationException(
-                    Resources.FormatUnexpectedException(
-                        typeof(DefaultTagHelperResolver).FullName,
-                        nameof(GetTagHelpersAsync)),
-                    exception);
+                throw new InvalidOperationException($"An unexpected exception occurred when invoking '{typeof(DefaultTagHelperResolver).FullName}.{nameof(GetTagHelpersAsync)}' on the Razor language service.", exception);
```

Comment:
Why not use the resource anymore? (Also line length)

Reply:
Since this moved from LanguageServices.Razor there's no more resource file and given we've in general been moving away from resources I just figured I'd take the plunge into resourceless land 😄